demos/index.html

```diff
@@ -8,37 +8,38 @@
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/overviewPanel/css/OverviewPanel.css" />
         <link rel="stylesheet" type="text/css" href="../src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css" />
 
-        <script type="text/javascript" src="../src/lib/infusion/infusion-custom.js"></script>
-        <script type="text/javascript" src="../src/lib/d3/d3.js"></script>
-
-        <script type="text/javascript" src="../src/lib/flocking/flocking-base.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/oscillators.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/math.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/envelopes.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/midi.js"></script>
-        <script type="text/javascript" src="../src/lib/flocking/ugens/scheduling.js"></script>
-
-        <script type="text/javascript" src="../src/js/valueBinding.js"></script>
-        <script type="text/javascript" src="../src/js/percentage.js"></script>
-        <script type="text/javascript" src="../src/js/transformations.js"></script>
-        <script type="text/javascript" src="../src/js/templateInjection.js"></script>
-        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
-        <script type="text/javascript" src="../src/js/totalRelaying.js"></script>
-        <script type="text/javascript" src="../src/js/legend.js"></script>
-        <script type="text/javascript" src="../src/js/pie.js"></script>
-        <script type="text/javascript" src="../src/js/pieChart.js"></script>
-        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
-        <script type="text/javascript" src="../src/js/utils.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntry.js"></script>
-        <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
-        <script type="text/javascript" src="../src/js/electricPianoBand.js"></script>
-        <script type="text/javascript" src="../src/js/unitDivisorSonificationStrategy.js"></script>
-        <script type="text/javascript"
-        <script type="text/javascript" src="../src/js/sonifier.js"></script>
-        <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
-
-        <script type="text/javascript" src="src/js/chartAuthoring-demo-overview.js"></script>
-        <script type="text/javascript" src="src/js/chartAuthoring-demo.js"></script>
+        <script  src="../src/lib/infusion/infusion-custom.js"></script>
```

Comment:
Please update to the most recent Infusion (2.0.0-dev.20160519T222603Z.754d2c6 is in npm and is pretty stable) since there have been a number of recent changes to model relay, etc. that may have caused some drift